### PR TITLE
Postman

### PR DESCRIPTION
**Pull request by alexdavcudriz**
## Summary
- Add list setters in models.
- Fix book model setters.
- Set id in update services.
## Trello Card
https://trello.com/c/I9Kk6Y0z/39-postman

### CODE REVIEW

File: src/main/java/wolox/training/services/BookService.java [hunk 1]
```diff
@@ -26,6 +26,7 @@ public Book save(Book book) {
 
     public Book update(Book book, Long id) {
         bookRepository.findById(id).orElseThrow(BookNotFoundException::new);
```
Comment (jdhiguita) [inline src/main/java/wolox/training/services/BookService.java:32]: Add validation to id vs book.getId() is same
```
if (book.getId() != id) {
            throw new BookIdMismatchException();	           
        }
```

File: src/main/java/wolox/training/services/BookService.java [hunk 2]
```diff
@@ -26,6 +26,7 @@ public Book save(Book book) {
...
+        book.setId(id);
```
Comment (jdhiguita) [inline src/main/java/wolox/training/services/BookService.java]: This is not necessary

File: src/main/java/wolox/training/services/UserService.java [hunk 3]
```diff
@@ -32,6 +32,7 @@ public User create(User user) {
 
     public User update(User user, Long id) {
         userRepository.findById(id).orElseThrow(BookNotFoundException::new);
```
Comment (jdhiguita) [inline src/main/java/wolox/training/services/UserService.java:38]: Add validation to id vs user.getId()
`if (user.getId() != id) {
            throw new UserIdMismatchException();	           
        }`

File: src/main/java/wolox/training/services/UserService.java [hunk 4]
```diff
@@ -32,6 +32,7 @@ public User create(User user) {
...
+        user.setId(id);
```
Comment (jdhiguita) [inline src/main/java/wolox/training/services/UserService.java]: This is not necessary

**Review by jdhiguita — COMMENTED** [review]
Good job! Some comments

**Review by facucompe — APPROVED** [review]

**Review by jdhiguita — APPROVED** [review]

**Pull request merged**